data/db_demo.yaml

```diff
@@ -890,6 +890,7 @@ shifts:
     name: "the Night Shift"
     shift_admins:
       - =testadmin
+    required_users_number: 6
```

Comment:
Is required or requested better here?

Reply:
I'm not so sure. It seems to be that it is a requirement, as otherwise the shift is incomplete. I'll let you decide you have more experience than me being on shift.